### ISSUE TITLE
Ports: Connections table

### ISSUE THREAD

**Issue by jlguillen** [SUNET/sri-front#236]
Ports - Missing "Connections", Should be 'filterable'.

**Comment by PaKZer0** [comment]
This means for each port table in SRI (like routers)

**Comment by mariahaider** [comment]
I don't see the 'filter' option.

**Comment by PaKZer0** [comment]
As Maria pointed out the filter is not there yet.

**Comment by PaKZer0** [comment]
@mariahaider This table would only hold one row isn't it? A port would only be connected to one cable, at least that's what we got from the code and talking to Johan.

If there's only one row we can't see the point on having a filter on this table.

**Comment by mariahaider** [comment]
Yeah, right, then it is not needed. I saw in our NI and thought it was missing :)

**Comment by mariahaider** [comment]
Everything else looks good except the 'Description' in 'Connected to' cable does not show its value.
